client/src/test/java/tds/session/SessionTest.java

```diff
@@ -85,16 +83,6 @@ public void shouldCrreateAGuestSession() {
     }
 
     @Test
-    public void shouldCreateAGuestSessionCaseInsensitive() {
```

Comment:
This doesn't appear to be related to the session type change.  Was this supposed to be removed?